### DISCUSSION TITLE
Managing Shared Albums

### DISCUSSION THREAD

**Discussion by pgalbavy** [photoprism/photoprism#3947] [Q&A] [answered]
Is there any way in the UI to manage shared albums?

What I mean is, as usage grows, I will have many shared albums and other objects. I would like one place to see what is shared and to be able to cancel some shares and so on. If this exists, I have failed to find it.

**Comment by graciousgrey** [comment, marked as answer]
This as well as many other improvements planned for sharing :)

- https://github.com/photoprism/photoprism/issues/1330
- https://github.com/photoprism/photoprism/issues?q=is%3Aissue+is%3Aopen+Sharing+in%3Atitle
